src/stop.js

```diff
@@ -0,0 +1,60 @@
+const stopASGs = require('./asgs/stopASGs');
+const listASGsToStop = require('./asgs/listASGsToStop');
+const tagASGs = require('./asgs/tagASGs');
+const listInstancesToStop = require('./instances/listInstancesToStop');
+const tagInstances = require('./instances/tagInstances');
+const stopInstances = require('./instances/stopInstances');
+
+function spinDownASGs() {
+  return listASGsToStop()
+    .then((stoppableASGs) => {
+      console.log(`Found the following ${stoppableASGs.length} instances to spin down...`);
+      if (stoppableASGs.length === 0) {
+        console.log('None! Moving on.');
+        return 'No ASGs to spin down';
+      }
+
+      stoppableASGs.forEach((asg) => {
+        console.log(asg.AutoScalingGroupName);
+      });
+      return tagASGs(stoppableASGs);
+    })
+    .then((taggedASGs) => {
+      console.log(`Finished tagging ASGs. Moving on to spin down ${taggedASGs.length} of them.`);
+      return stopASGs(taggedASGs);
```

Comment:
Ahh yes, the export for `stopASGs` looks like it is just meant to be the function instead of an object export, will fix this up.